doc/api/deprecations.md

```diff
@@ -634,6 +634,17 @@ Type: Runtime
 
 *Note*: change was made while `async_hooks` was an experimental API.
 
+<a id="DEP00XX"></a>
```

Comment:
yeah, it's better to assign when landing... or likely better yet to handle it like the `REPLACEME` metadata eventually. The reason is because if we end up with more than one deprecation PR at any given time, there's a greater chance of accidentally duplicating the codes or getting them out of order somehow.